src/views/compute/DeployVM.vue

```diff
@@ -1409,7 +1409,6 @@ export default {
           this.$router.back()
         }).catch(error => {
           this.$notifyError(error)
-        }).finally(() => {
```

Comment:
Shouldn't we reset in finally? Otherwise it will reset only on error. Or, is the logic that on VM deployment we'll change view so this is not necessary?

Reply:
On success, it goes back to the previous page. If we keep it in finally, the button will show up for a second before the page redirects